internal/compiler-interface/src/main/contraband/incremental.json

```diff
@@ -124,29 +117,6 @@
           ]
         },
         {
-          "name": "antStyle",
```

Comment:
The scala build currently "relies" on antStyle, for some definition of "relies":

* https://github.com/scala/scala/blob/5f1a638cf5f3b81fd3d8b4fa92a5a52ac5733517/build.sbt#L884
* https://github.com/scala/scala/blob/5f1a638cf5f3b81fd3d8b4fa92a5a52ac5733517/README.md#incremental-compilation

Are we just going to ignore them? Or has this been cleared with them on some back-channel?